### ISSUE TITLE
Cursor Quality Problem

### ISSUE THREAD

**Issue by Tahinli** [aleiepure/devtoolbox#49]
**Describe the issue**
Hi,
Firstly it's a great program thanks for your effort.

My cursor is getting weird when I swipe it onto program. It loses quality and seems bigger.

**To Reproduce**
Steps to reproduce the behaviour:
1. Open devtoolbox
2. Move your cursor on it
3. See the problem visually

**Expected behavior**
My cursor shouldn't be seen like this

**Screenshots**

https://github.com/user-attachments/assets/cf6c06eb-742d-4e38-bdb9-48686a3a2436

**Operating system (please complete the following information):**
 - Distro: Fedora 40
 - Version: 1.2
 - Desktop Environment: KDE Plasma
 - Xorg/Wayland: Wayland

**Comment by aleiepure** [comment]
I don't think this is an issue with this app perse, but it is due to the different technologies the GNOME and KDE desktop environments. 
For instance, I am running GNOME and I have the same issue with most electron apps (vs code, spotify, discord, etc).
I will look further into it.

**Comment by aleiepure** [comment]
Have a read here https://blogs.kde.org/2024/10/09/cursor-size-problems-in-wayland-explained/ and try installing `xdg-desktop-portal-gtk` as indicated in the article.

**Comment by Tahinli** [comment]
I've already had xdg-desktop-portal-gtk
